### DISCUSSION TITLE
Red6 Question on PdfCompare and no difference

### DISCUSSION THREAD

**Discussion by uday0804** [red6/pdfcompare#140] [Q&A]
Does red6 generate differences percentages and files if there are no differeneces?

**Comment by finsterwalder** [comment]
When there are no differences, then you don't get a difference percentage. Or rather a percentage of 0%.
You can decide, whether pages without differences should go into the result.
addEqualPagesToResult=true (default true)
So normally pages that didn't change are also in the result.

**Reply by uday0804** [reply]
![image](https://github.com/red6/pdfcompare/assets/38447637/5f0127a2-5948-43bd-b851-6ac603da346d)

I think that is the reason I am seeing this error. Is there a way to handle it better way?

**Reply by finsterwalder** [reply]
No. I think that error means, you are actually passing an empty file name to PdfCompare.
When you pass two files, that are identical, you will not get an error. You will get a Compare result that shows 0% difference.
